games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -96,7 +96,7 @@ INSTALL()
 {
 	make -C build install
  	mkdir -p $appsDir
-	mv $binDir/fgfs $appsDir/FlightGear
+ 	mv $binDir/fgfs $appsDir
```

Comment:
Oops, there's a white space before the tab. It's my fault, sorry.. (There was an unwanted space in my comment.)
```diff
- 	mv $binDir/fgfs $appsDir
+	mv $binDir/fgfs $appsDir
```